lib/xboxrt/assert.h

```diff
@@ -0,0 +1 @@
+#define assert(x)
```

Comment:
This is bad, because it allows for `assert(1) assert(0)` to work, without a semicolon between them.

Reply:
Can be removed - fixed upstream.